apps/web-dgf/.env.example

```
@@ -8,3 +8,11 @@ NEXT_PUBLIC_GA_MEASUREMENT_ID=
 FOXY_CART_SIGNING_KEY=
 SANITY_API_READ_TOKEN=
 SANITY_API_WRITE_TOKEN=
+CONSTANT_CONTACT_CLIENT_ID=
+CONSTANT_CONTACT_CLIENT_SECRET=
+CONSTANT_CONTACT_REFRESH_TOKEN=
+CONSTANT_CONTACT_LIST_ID=
+CONSTANT_CONTACT_REDIRECT_URI=
+CONSTANT_CONTACT_TOKEN_STORE_KEY=constant-contact:web-dgf
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CONSTANT_CONTACT_TOKEN_STORE_KEY key should go before the FOXY_CART_SIGNING_KEY key

(UnorderedKey)